Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/local/source/LocalStatisticsCache.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.statistics.local.source
+
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+import de.rki.coronawarnapp.statistics.Statistics
+import de.rki.coronawarnapp.statistics.local.FederalStateToPackageId
+import timber.log.Timber
+import java.io.File
+
+class LocalStatisticsCache @AssistedInject constructor(
+    @Statistics cacheDir: File,
+    @Assisted private val stateForCache: FederalStateToPackageId,
+) {
+
+    private val cacheFolder = File(cacheDir, "cache_raw_local")
+    private val cacheFile = File(cacheFolder, stateForCache.toString())
```

Comment:
Please use something more explicit than `.toString()`, could be too easily accidentally affected by refactoring.